metaflow/plugins/aip/argo_client.py

```diff
@@ -88,7 +88,12 @@ def trigger_workflow_template(self, name: str, parameters: Optional[Dict] = None
         body = {
             "apiVersion": "argoproj.io/v1alpha1",
             "kind": "Workflow",
-            "metadata": {"generateName": name + "-"},
+            "metadata": {
+                "generateName": name + "-",
+                "labels": {
+                    "zodiac.zillowgroup.net/product": "batch"
+                }
```

Comment:
How about component of WFSDK?
Otherwise, we have no way to discern between WFSDK pods and Spark executors.

Reply:
The component label would ideally be applied to the pods...  Otherwise the webhook will have to discern using the argo label.

Reply:
I think Taylor has this here for the validating webhook he created where he checks the Workflow custom resource.